assets/queries/ansible/gcp/cloud_mysql_instance_with_local_infile_on/query.rego

```diff
@@ -1,38 +1,22 @@
 package Cx
 
+import data.generic.ansible as ansLib
+
 CxPolicy[result] {
 	document := input.document[i]
-	tasks := getTasks(document)
-	task := tasks[t]
+	task := ansLib.getTasks(document)[t][k]
```

Comment:
Replace with: `task := ansLib.getTasks(document)[t]`
The last iteration variable was necessary because the positive and negative samples use the `tasks` field, and when this field is used, the `getTasks` function doesn't work correctly. This will be fixed later. You'll have to remove the `tasks` field from the samples, so it can pass the tests.